tpot/tpot.py

```diff
@@ -261,16 +267,29 @@ def pareto_eq(ind1, ind2):
 
             self.hof = tools.ParetoFront(similar=pareto_eq)
 
-            stats = tools.Statistics(lambda ind: ind.fitness.values[1])
-            stats.register('Minimum score', np.min)
-            stats.register('Average score', np.mean)
-            stats.register('Maximum score', np.max)
+            # stats = tools.Statistics(lambda ind: ind.fitness.values[1])
+            # stats.register('Minimum score', np.min)
+            # stats.register('Average score', np.mean)
+            # stats.register('Maximum score', np.max)
 
             verbose = (self.verbosity == 2)
 
-            pop, _ = algorithms.eaSimple(population=pop, toolbox=self._toolbox, cxpb=self.crossover_rate,
+            # Start the progress bar
+            num_evaluations = self.population_size * (self.generations + 1)
+            self.pbar = tqdm(total=num_evaluations, unit='pipeline',
+                             disable=(not verbose), desc='GP Progress')
+
+            with warnings.catch_warnings():
+                # Ignore warnings
+                warnings.simplefilter('ignore', category=UserWarning)
```

Comment:
I see. In that case, I think it's better to discover what specific operators are throwing those warnings and -- if we decide that the warnings should be ignored -- put those "ignore warning" wrapper around the specific calls that throw that warning, as we do in e.g. `select_kbest()`.

In any case, this bit of code should be in a separate issue/PR IMO. Please remove this code from this PR, raise an issue for this concern, and we'll decide on course of action there.